homeassistant/components/daikin/sensor.py

```diff
@@ -135,9 +144,9 @@ class DaikinPowerSensor(DaikinSensor):
     def state(self):
         """Return the state of the sensor."""
         if self._device_attribute == ATTR_TOTAL_POWER:
-            return round(self._api.device.current_total_power_consumption, 3)
+            return round(self._api.device.current_total_power_consumption, 2)
         if self._device_attribute == ATTR_COOL_ENERGY:
-            return round(self._api.device.last_hour_cool_energy_consumption, 3)
+            return round(self._api.device.last_hour_cool_power_consumption, 2)
```

Comment:
this should be reverted back to energy

Reply:
Please also update the description

Reply:
This is the `pydaikin` API

Reply:
I could push a new pydaikin versioniif you like..

Reply:
@mlemainque are you preparing code for the pydaikin lib or should I?

Reply:
Is it really necessary @dgomes? :)

Reply:
@mlemainque is this enough?

```diff
diff --git a/pydaikin/daikin_base.py b/pydaikin/daikin_base.py
index 404721c..cb868f1 100644
--- a/pydaikin/daikin_base.py
+++ b/pydaikin/daikin_base.py
@@ -208,9 +208,9 @@ class Appliance(DaikinPowerMixin):  # pylint: disable=too-many-public-methods
             )
             data.append(('cool_today', self.energy_consumption(ATTR_COOL, TIME_TODAY)))
             data.append(('heat_today', self.energy_consumption(ATTR_HEAT, TIME_TODAY)))
-            data.append(('total_power', self.current_total_power_consumption))
-            data.append(('cool_power', self.last_hour_cool_power_consumption))
-            data.append(('heat_power', self.last_hour_heat_power_consumption))
+            data.append(('total_power', self.current_total_energy_consumption))
+            data.append(('cool_power', self.last_hour_cool_energy_consumption))
+            data.append(('heat_power', self.last_hour_heat_energy_consumption))
         if file.tell() == 0:
             file.write(','.join(k for k, _ in data))
             file.write('\n')
@@ -238,9 +238,9 @@ class Appliance(DaikinPowerMixin):  # pylint: disable=too-many-public-methods
             data.append(
                 f'heat_today={self.energy_consumption(ATTR_HEAT, TIME_TODAY):.01f}kWh'
             )
-            data.append(f'total_power={self.current_total_power_consumption:.02f}kW')
-            data.append(f'cool_power={self.last_hour_cool_power_consumption:.01f}kW')
-            data.append(f'heat_power={self.last_hour_heat_power_consumption:.01f}kW')
+            data.append(f'total_power={self.current_total_energy_consumption:.02f}kW')
+            data.append(f'cool_power={self.last_hour_cool_energy_consumption:.01f}kW')
+            data.append(f'heat_power={self.last_hour_heat_energy_consumption:.01f}kW')
         print('  '.join(data))
 
     def represent(self, key):
@@ -349,28 +349,28 @@ class Appliance(DaikinPowerMixin):  # pylint: disable=too-many-public-methods
         return self._parse_number('shum')
 
     @property
-    def current_total_power_consumption(self):
+    def current_total_energy_consumption(self):
         """Return the current total power consumption in kW."""
         # We tolerate a 50% delay in consumption measure
-        return self.current_power_consumption(
+        return self.current_energy_consumption(
             mode=ATTR_TOTAL, exp_diff_time_margin_factor=0.5
         )
 
     @property
-    def last_hour_cool_power_consumption(self):
+    def last_hour_cool_energy_consumption(self):
         """Return the last hour cool power consumption of a given mode in kW."""
         # We tolerate a 5 minutes delay in consumption measure
-        return self.current_power_consumption(
+        return self.current_energy_consumption(
             mode=ATTR_COOL,
             exp_diff_time_value=timedelta(minutes=60),
             exp_diff_time_margin_factor=timedelta(minutes=5),
         )

 
     @property
-    def last_hour_heat_power_consumption(self):
+    def last_hour_heat_energy_consumption(self):
         """Return the last hour heat power consumption of a given mode in kW."""
         # We tolerate a 5 minutes margin in consumption measure
-        return self.current_power_consumption(
+        return self.current_energy_consumption(
             mode=ATTR_HEAT,
             exp_diff_time_value=timedelta(minutes=60),
             exp_diff_time_margin_factor=timedelta(minutes=5),
diff --git a/pydaikin/power.py b/pydaikin/power.py
index 08ce689..522543c 100644
--- a/pydaikin/power.py
+++ b/pydaikin/power.py
@@ -149,7 +149,7 @@ class DaikinPowerMixin:
         _LOGGER.error('Impossible energy consumption measure of %s', mode)
         return None
 
-    def current_power_consumption(  # pylint: disable=too-many-branches
+    def current_energy_consumption(  # pylint: disable=too-many-branches
         self,
         mode=ATTR_TOTAL,
         exp_diff_time_value=None,
```

Reply:
The `current_power` should remain the unchanged. https://bitbucket.org/mustang51/pydaikin/pull-requests/45/rename-power-to-energy-consumption